core/renderers/common/marker_svg.js

```diff
@@ -89,34 +98,34 @@ Blockly.blockRendering.MarkerSvg = function(workspace, constants, marker) {
  * The name of the CSS class for a cursor.
  * @const {string}
  */
-Blockly.blockRendering.MarkerSvg.CURSOR_CLASS = 'blocklyCursor';
+MarkerSvg.CURSOR_CLASS = 'blocklyCursor';
```

Comment:
`CURSOR_CLASS`, `MARKER_CLASS`, and `HEIGHT_MULTIPLIER` can all be module-internal and just declared as const.

@alschmiedt to confirm that these are actually const and that we don't set them in weird ways, and also that they can become private.

Reply:
Yes, can confirm they should be module-internal.